urbanaccess/gtfs/network.py

```diff
@@ -69,6 +71,15 @@ def create_transit_net(
         directory to save the HDF5 file
     save_filename : str, optional
         name to save the HDF5 file as
+    timerange_pad: int, optional
```

Comment:
Seems like it would make sense for this to be a float, to allow for partial hours if needed?

Reply:
That's a great idea @smmaurer! However, I think that the majority of people would be confused to type 2.25 instead of 2:15:00. so I'm thinking we should change this to a string and modify the code a bit to convert the string to a float. Any thoughts @sablanchard ?

Reply:
If you want to support partial hours then I agree with @bouzaghrane to use a string instead, it will also be consistent with how time is specified in other functions anyway. If agreed, I can add that.

Reply:
I agree, this sounds like a good approach!

Reply:
this has been added here: https://github.com/UDST/urbanaccess/pull/87/commits/07540c0d5e75e27808f32d9ca8d15459b983d0f7